app/assets/javascripts/media_player_wrapper/mejs4_helper_markers.es6

```diff
@@ -301,6 +301,13 @@ class MEJSMarkersHelper {
         $('#markers').replaceWith(response);
         // Add event listeners to newly created row
         t.addMarkersTableListeners();
+        // hide marker sections if no markers (first row is header)
+        if ($('#markers').find('tr').length == 1) {
```

Comment:
In JS, should always (99% of the time) use `===` instead of `==`.  Just noticed this on CodeClimate.